ui/page/governance/governance_page.go

```diff
@@ -99,6 +99,13 @@ func (pg *Page) HandleUserInteractions() {
 }
 
 func (pg *Page) Layout(gtx C) D {
+	if pg.Load.GetCurrentAppWidth() <= gtx.Dp(values.StartMobileView) {
+		return pg.layoutMobile(gtx)
+	}
+	return pg.layoutDesktop(gtx)
+}
+
+func (pg *Page) layoutDesktop(gtx layout.Context) layout.Dimensions {
 	if !pg.isGovernanceFeatureEnabled() {
 		return components.UniformPadding(gtx, pg.splashScreenLayout)
 	}
```

Comment:
```suggestion

if !pg.isGovernanceFeatureEnabled() {
	if pg.Load.GetCurrentAppWidth() <= gtx.Dp(values.StartMobileView) {
	    return components.UniformMobile(gtx, false, pg.splashScreenLayout)
	}
		return components.UniformPadding(gtx, pg.splashScreenLayout)
}
```

Reply:
we should leave as is, further details here https://github.com/planetdecred/godcr/pull/971#discussion_r904149266